types/vimeo__player/index.d.ts

```diff
@@ -34,7 +34,7 @@ export class Player {
 
     on(event: EventName, callback: EventCallback): void;
     off(event: EventName, callback?: EventCallback): void;
-    loadVideo(id: number): VimeoPromise<number, TypeError | PasswordError | PrivacyError | Error>;
+    loadVideo(id: number | string): VimeoPromise<number, TypeError | PasswordError | PrivacyError | Error>;
```

Comment:
this change is ok, the source supports and option object as well.
The only problem is, should it be versioned? (does it really work with version 2.10, the docs/readme change is newer)

Reply:
Good point. It's hard to say. Shall I just bump the version to match the docs?